Description: Take debian multiarch into account when defining the include path.
Origin: upstream, http://cgit.freedesktop.org/gstreamer/qt-gstreamer/commit/?id=c741563913142aa8a4ba87d0ae3ae872b6487004

diff --git a/cmake/modules/FindQtGStreamer.cmake b/cmake/modules/FindQtGStreamer.cmake
index 4c7d3f4..54e75f6 100644
--- a/cmake/modules/FindQtGStreamer.cmake
+++ b/cmake/modules/FindQtGStreamer.cmake
@@ -60,14 +60,20 @@ else()
                          QTGSTREAMER_UTILS_LIBRARY QTGSTREAMER_INCLUDE_DIR)
     else()
         # Targets file found. Use imported QtGStreamer target and relative include path.
-        # We assume that this file has been installed in $PREFIX/lib/QtGStreamer/,
+        # We assume that this file has been installed in $PREFIX/lib(64)/QtGStreamer/,
+        # or $PREFIX/lib/<arch_triplet>/QtGStreamer, or $PREFIX/lib(64)/cmake/QtGStreamer/,
+        # or $PREFIX/lib/<arch_triplet>/cmake/QtGStreamer/,
         # so the include path should evaluate to $PREFIX/include/QtGStreamer
         include(${_QTGSTREAMER_TARGETS_FILE})
         set(QTGLIB_LIBRARY QtGLib)
         set(QTGSTREAMER_LIBRARY QtGStreamer)
         set(QTGSTREAMER_UI_LIBRARY QtGStreamerUi)
         set(QTGSTREAMER_UTILS_LIBRARY QtGStreamerUtils)
-        get_filename_component(QTGSTREAMER_INCLUDE_DIR "${_QTGSTREAMER_CONFIG_DIR}/../../include/QtGStreamer" ABSOLUTE)
+        find_path(QTGSTREAMER_INCLUDE_DIR QGst/Global
+                  PATHS "${_QTGSTREAMER_CONFIG_DIR}/../../include/QtGStreamer"
+                        "${_QTGSTREAMER_CONFIG_DIR}/../../../include/QtGStreamer"
+                        "${_QTGSTREAMER_CONFIG_DIR}/../../../../include/QtGStreamer"
+                  NO_DEFAULT_PATH)
     endif()
 endif()
 
